server/api/support.py

```diff
@@ -89,7 +100,11 @@ def list_active_elections():
             )
         )
         .join(Organization)
-        .order_by(Organization.name, Election.audit_name)
+        .join(
+            ActivityLogRecord,
+            ActivityLogRecord.info["base"]["election_id"].as_string() == Election.id,
+        )
+        .order_by(ActivityLogRecord.timestamp.desc())
```

Comment:
I don't love that I order the results for active audits in the server endpoint, and order the audits on the organization level in the client. However it seemed to make way more sense to order this specific endpoint here and not have the ActivityLogRecord knowledge need to leave this function. Otherwise for the organization view sorting by creation date felt more like a choice of that particular page in the UI to sort that way. But I could be swayed to make it consistent either way.

Reply:
This feels fine to me!

I do think you could potentially simplify this query now that we're joining on `ActivityLogRecord` to also do the filtering after the join, rather than in a subquery. I don't think it will make a huge difference but might just end up a tad simpler